tests/unit/type/hash-field-expire.tcl

```diff
@@ -599,6 +599,20 @@ start_server {tags {"external:skip needs:debug"}} {
             wait_for_condition 30 10 { [r exists myhash2] == 0 } else { fail "`myhash2` should be expired" }
         }
 
+        test "Test RENAME hash that had HFEs but not during the rename ($type)" {
+            r del h1
+            r hset h1 f1 v1 f2 v2
+            r hpexpire h1 1 FIELDS 1 f1
+            after 20
+            r rename h1 h1_renamed
+            assert_equal [r exists h1] 0
+            assert_equal [r exists h1_renamed] 1
```

Comment:
maybe we want to use tags? when running with proxy

Reply:
in the CE tests/README.md and tests/support/test.tcl, we don't have proxy reference (only external / cluster).
in the RE fork we do have them, but we lately started just adding them to an externally maintained proxy skip file.